rosdep/base.yaml

```diff
@@ -792,6 +794,13 @@ gazebo7:
   gentoo: [sci-electronics/gazebo]
   slackware: [gazebo]
   ubuntu: [gazebo7]
+gazebo9:
```

Comment:
These rules are a little over broad. Gazebo9 isn't available on all versions of these distros. Can we make this a little bit more specific so that we don't have rosdep trying to install gazebo9 on unsupported platforms?

Reply:
Also where is gazebo9 available from? My machine only shows as available from packages.osrfoundation.org (xenail) and rmadison only shows it for bionic and cosmic. If it's not in the default sources + ROS we shouldn't have it here.

Reply:
Ouch, Gazebo9 is not available in all the distributions listed here, you are right. It is available for Xenial from packages.osrfoundation.org, and Ubuntu servers for Bionic. In Debian, it is available in Unstable and Testing. In Gentoo from the official repo. In Arch from the User repositories.